src/cli/start.rs

```diff
@@ -74,6 +74,12 @@ pub struct StartCommandArguments {
 		requires = "username"
 	)]
 	password: Option<String>,
+	#[arg(
```

Comment:
```suggestion
        # TODO(gguillemas): Remove this arg once the legacy basic auth is deprecated in v2.0.0, because it will be enabled by default
	#[arg(
```